pyproject.toml

```diff
@@ -23,18 +23,16 @@ lnurl = "0.4.2"
 psycopg2-binary = "2.9.7"
 pydantic = "1.10.17"
 pyqrcode = "1.2.1"
-shortuuid = "1.0.11"
-sqlalchemy = "1.3.24"
-sqlalchemy-aio = "0.17.0"
+shortuuid = "1.0.13"
+sqlalchemy = "1.4.52"
 sse-starlette = "1.6.5"
-typing-extensions = "4.8.0"
 uvicorn = "0.23.2"
 uvloop = "0.19.0"
 websockets = "11.0.3"
 loguru = "0.7.2"
 grpcio = "1.62.2"
 protobuf = "4.24.3"
-pyln-client = "23.8"
+pyln-client = "24.5"
```

Comment:
do not upgrade this without adjusting the code - i tried to change it earlier and it failed because they changed the API

Reply:
i found out myself aswell :),  :dagger: